eoscostarica/eosio-components
# Feat/ricardian contract params and styles adjustments

### Add params to Ricardian Contract and BP.json Generator

### What does this PR do?

- Allow to pass params to link to abi and actions in the Ricardian Contract Component
- Apply styles adjustments to the BP.json Generator
- Allow to pass additional node types in the BP.json Generator Component
- Show an error when the BP.json has a wrong value
- Update dependencies with no breaking changes

### Steps to test

1. Run the project locally
2. Go to Ricardian Contract
3. Try to set a block explorer url with abi or actions param
4. Go to BP.json Generator
5. Scroll down until find "Add node" button and click it
6. Check the modal 

### Screenshots

# Ricardian Contract Component

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e6155829-0e78-4f46-80b8-e823b245ba5b)

# BP.json Generator Component

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/abb96b5a-cc09-4fc2-a2a0-7c5889146192)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/1b5e07a7-cff4-47f5-956c-540b3155ed81)

## Additional node types as params

![imagen](https://github.com/eoscostarica/eosio-components/assets/66583677/3e8f7e93-a447-41a1-a167-5c2672b798a7)

## Allow to add any node feature or select one from the standard

![imagen](https://github.com/eoscostarica/eosio-components/assets/66583677/3a52bfbc-b5d2-467a-acd2-b919c21a2d59)

# Error Modal

![imagen](https://github.com/eoscostarica/eosio-components/assets/66583677/b01114c6-fa56-4b6b-8ded-a08baea2c1c3)
